monkey/infection_monkey/post_breach/tests/actions/test_users_custom_pba.py

```diff
@@ -0,0 +1,152 @@
+import pytest
```

Comment:
Unit tests are nice, but not a silver bullet. There's an antipattern in TDD called the "mockery", when most of the functionality is mocked. I don't suggest refactoring the whole pba's to solve it, but I can see BB tests bringing more value here and still being trivial enough to implement (command execution at least). Other PBA's should be tested with BB's IMO.